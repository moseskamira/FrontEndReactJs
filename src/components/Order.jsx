import React, { Component } from 'react';


export class Order extends Component {


  render() {
    return (
      <div>
        <section>
        <div>
            <div>
            <h2> <strong><center>WELCOME TO FAST FOOD FAST </center></strong></h2>
            <h5> <strong><center>ENJOY THE SERVICE </center></strong></h5>
            </div>
        </div>
        </section>
      </div>
    );
  }
}

export default Order
