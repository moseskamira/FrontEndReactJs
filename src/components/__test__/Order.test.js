import React from "react";
import { shallow } from "enzyme";
import { Order } from "../Order";

describe("Order Component", () => {
  let wrapper;
  const historyMock = { push: jest.fn() };
  const mockCreate = jest.fn();

  beforeEach(() => {
    wrapper = shallow(<Order />);
  });
  afterEach(() => {
    mockCreate.mock.calls = [];
  });
  describe("Testing the Order component", () => {
    it("should render without crashing", () => {
      expect(wrapper).toMatchSnapshot();
    });
  });
});
