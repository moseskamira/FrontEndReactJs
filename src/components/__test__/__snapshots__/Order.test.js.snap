// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Order Component Testing the Order component should render without crashing 1`] = `ShallowWrapper {}`;
